src/Reader/readpar.f90

```diff
@@ -381,6 +371,24 @@ SUBROUTINE readpar_equations(EQN,DISC,SOURCE,IC, IO)
         logError(*) 'Choose 0, 1 as adjoint wavefield assumption. '
         STOP
       END SELECT
+    
+    EQN%Anisotropy = Anisotropy
+    SELECT CASE(Anisotropy)
```

Comment:
Is there a reason why you used a select and not an if statement?

Reply:
To make it go along with the code above.